packages/dendron-plugin-views/src/components/DendronApp.tsx

```diff
@@ -87,7 +87,9 @@ function DendronVSCodeApp({ Component }: { Component: DendronComponent }) {
           await ideDispatch(engineSlice.syncNote({ ...workspace, note }));
         }
         logger.info({ ctx, msg: "setNoteActive:pre" });
-        ideDispatch(ideSlice.actions.setNoteActive(note));
+        // If activeNote is in the data payload, set that as active note. Otherwise default to changed note
```

Comment:
Can you move (or copy) this comment into `typesv2` for `activeNote` - basically to explain the logic to someone who is going to call the API of when they should set activeNote and not. 

Also - could you add a `TODO:` to that comment as well that we should split this up into a separate command beyond `OnDidChangeActiveTextEditor`? I.e. `onNoteStateChanged` - I think we should make that change soon.

Reply:
Created new task for TODO [[Split up Ondidchangeactivetexteditordata|dendron://private/task.2022.02.23.split-up-ondidchangeactivetexteditordata]]